src/Workspaces/Core/Portable/PatternMatching/PatternMatchKind.cs

```diff
@@ -24,9 +24,16 @@ internal enum PatternMatchKind
         Prefix,
 
         /// <summary>
-        /// The pattern was a substring of the candidate string, but in a way that wasn't a CamelCase match.
+        /// The pattern was a substring of the candidate string, but in a way that wasn't a CamelCase match.  The
+        /// pattern had to have at least one non lowercase letter in it, and the match needs to be case sensitive.
```

Comment:
Please have some examples in the comment for each of the new substring matching.

Reply:
Done.